python-sdk/nuscenes/predict/tests/test_mtp.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import torch
+from torchvision.models.resnet import BasicBlock, Bottleneck
+from nuscenes.predict.models import mtp
+
+class TestBackBones(unittest.TestCase):
+
+    def count_layers(self, model):
+        if isinstance(model[4][0], BasicBlock):
+            n_convs = 2
+        elif isinstance(model[4][0], Bottleneck):
+            n_convs = 3
```

Comment:
Suggest to have an else case here in case none of the above is true.

Reply:
Done!